README.md

```diff
@@ -109,6 +109,26 @@ NormalMode and CheckMode are still used. Generally speaking:
 * sub-section check your answers pages use NormalMode
 * section check your answers pages use CheckMode
 
+### Tampermonkey Scripts
+Tampermonkey scripts have been developed for automating the answering of questions within the departure frontends. The scripts can be found here: https://github.com/hmrc/manage-transit-movements-departure-frontend/tree/CTCP-5537/tamperMonkey
```

Comment:
Could use a relative file path here

`The scripts can be found [here](tamperMonkey).`